test/integration/targets/win_iis_website/tasks/tests.yml

```diff
@@ -0,0 +1,503 @@
+---
+- name: try and create web site without path
+  win_iis_website:
+    name: '{{test_iis_website_name}}'
+    state: started
+  register: create_fail_no_path
+  failed_when: create_fail_no_path.msg != 'physical_path is required to be set when creating a new website'
+
+- name: try and create web site with invalid path
+  win_iis_website:
+    name: '{{test_iis_website_name}}'
+    state: started
+    physical_path: T:\fakepath
+  register: create_fail_invalid_path
+  failed_when: "create_fail_invalid_path.msg != 'physical_path is not a valid path: T:\\\\fakepath'"
```

Comment:
I'll give it a try, I believe I've come across issues with the `:` in the middle but I don't see how it is an actual issue.